src/shared/components/post/post.tsx

```diff
@@ -1507,4 +1522,109 @@ export class Post extends Component<PostRouteProps, PostState> {
     // Update the moderators
     // TODO: update GetCommunityResponse?
   }
+
+  duplicatesLine() {
+    if (this.state.postRes.state === "success") {
+      const dupes = this.state.postRes.data.cross_posts;
+
+      if (dupes && dupes.length > 0) {
+        const myUserInfo =
+          this.state.myUserInfoRes.state === "success"
+            ? this.state.myUserInfoRes.data
+            : undefined;
+        return (
+          <div className="d-flex flex-wrap">
```

Comment:
Instead of flex, I would use bootstrap columns, and probably `col-sm-12 col-md-6 col-lg-3`

Reply:
Changed although it doesnt make much difference.